jazz_logs/index.js

```diff
@@ -88,14 +88,27 @@ module.exports.handler = (event, context, cb) => {
 
 			querys.push(utils.setQuery("servicename", service));
 			querys.push(utils.setQuery("environment", env));
-			querys.push(utils.setQuery("log_level", logType));
 
 			//Query to filter Control messages
 			querys.push(utils.setQuery("!message", "START*"));
 			querys.push(utils.setQuery("!message", "END*"));
 			querys.push(utils.setQuery("!message", "REPORT*"));
 
-			logger.info("QueryObj: "+ JSON.stringify(querys));
+			if (logType) {
+				var log_type_config = [];
+				_.forEach(config.LOG_LEVELS, function(value, key) {
```

Comment:
consider using ES5 native function. you don't need lodash here

Reply:
Replaced Lodash forEach with native for loop.